tests/Feature/CDashTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
```

Comment:
The tests defined in this directory aren't getting run through ctest, right? Is that something that would be easy to add?